packages/i/i2c-tools/pspec_x86_64.xml

```diff
@@ -49,20 +50,22 @@
 </Description>
         <PartOf>programming.devel</PartOf>
         <RuntimeDependencies>
-            <Dependency release="3">i2c-tools</Dependency>
+            <Dependency release="4">i2c-tools</Dependency>
         </RuntimeDependencies>
         <Files>
             <Path fileType="header">/usr/include/i2c/smbus.h</Path>
+            <Path fileType="library">/usr/lib64/libi2c.a</Path>
```

Comment:
Nope, it didn't work. A static lib has crept in; this wasn't here before.
Please put that step beck on the install stage.